android/src/main/jni/CMakeLists.txt

```diff
@@ -0,0 +1,70 @@
+cmake_minimum_required(VERSION 3.13)
+set(CMAKE_VERBOSE_MAKEFILE on)
+
+set(RNS_ANDROID_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../..)
+set(RNS_COMMON_DIR ${RNS_ANDROID_DIR}/../common/cpp)
+set(RNS_GENERATED_DIR ${RNS_ANDROID_DIR}/build/generated)
+set(RNS_GENERATED_JNI_DIR ${RNS_GENERATED_DIR}/source/codegen/jni)
+set(RNS_GENERATED_REACT_DIR ${RNS_GENERATED_JNI_DIR}/react/renderer/components/rnscreens)
```

Comment:
Would it be possible to extract `rnscreens` to a variable so that library authors could change only that variable for their usecase?

Reply:
I guess I could do it, but it comes down to simply using find & replace option in code editor. I will, however refine variable names and do some cleanup.

Reply:
So I went with your suggestion. All it takes now is to change `LIB_LITERAL` constant. However some further customisation may be required as you noticed in RNSVG (adding custom dependencies etc.)